share/mrdocs/addons/generator/asciidoc/partials/signature/using.adoc.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if (eq symbol.class "namespace")}}
+using namespace {{#if symbol.qualifier}}{{>name-info symbol.qualifier}}::{{/if}}{{symbol.name}}
+{{else}}
+using {{#if (eq symbol.class "typename")}}typename {{/if}}{{#if (eq symbol.class "enum")}}enum {{/if}}{{#if symbol.qualifier}}{{>name-info symbol.qualifier}}::{{/if}}{{symbol.name}}{{#if symbol.symbols.length}} = {{>name-info symbol.symbols.[0]}}{{/if}}
```

Comment:
This shouldn't show the named symbols -- remove `{{#if symbol.symbols.length}} = {{>name-info symbol.symbols.[0]}}{{/if}}`.